ui/admin/app/controllers/scopes/scope/credential-stores/index.js

```diff
@@ -55,7 +55,6 @@ export default class ScopesScopeCredentialStoresIndexController extends Controll
    */
   @action
   applyFilter(paramKey, selectedItems) {
-    console.log('appy filter', paramKey, selectedItems);
```

Comment:
woops i left this behind, thx for cleaning it up!

Reply:
No worries. I missed it as well during the review.